Certification standard/main.html

```diff
@@ -0,0 +1,221 @@
+
+    </section>
+
+    <section id='sotd'>
+      This is a draft for the first version of the Cardano Certification Level Standards. 
+
+      The Working Group meets every two weeks on [Discord](https://discord.com/channels/1029837832350605343/1030116075532206101).
+
+      You can send questions on this [Discord Channel](https://discord.com/channels/1029837832350605343/1030116075532206101) or as [GitHub Issues]("https://github.com/input-output-hk/Certification-working-group"). 
+
+      [TODO:
+        How do we work on this document?
+        How are section added/removed?
+        How is the content agreed on and validated?
+        ?
+        ]
+    </section>
+
+    <section id="introduction" class="informative">
+      <h2>Introduction</h2>
+
+      This document is XXX.
+
+
+      [Todo:
+      - What is certification? What certification is NOT?
+      - Who could/should be seeking certifications?
+      - How to contribute? How to add vulnerabiities?
+      - Process for new versions?
+      - Define the syntax and semantics of the requirements/certification objectives
+      - Laws and other applicable standards? GDPR? MiCA?
+      ]
+
+    </section>
+
+    <section id='ccwg-conformance'>
+      <h2> Certification process</h2> 
+
+      <section>
+        <h3> What is certified? </h3>
+
+        A certification refers to a DApp, which means the source code of the on-chain component of a smart contract or several smart contracts, along the UPLC compiled from the source code and a given parametrisation.
+
+        [Discussion: Do we want to certify Dapps with general parameters or only instantions of such Dapps?]
+      </section>
+      <section>
+        <h3> On-chain certificates </h3>
+        
+        Say something about our CIP on certification metadata.
+      </section>
+      <section>
+        <h3>Who can issue certificates ? </h3>
+
+        This section XXX
+        
+        [Todo:
+        - Do we keep a list of trusted issuer? Can anyone issue certificates?
+        - How do we join the list?
+        - How do we remove someone from the list?
+        ]
+      </section>
+    </section>
+
+    <section id='certlevels'>
+      <h2>Certification Levels</h2>
+
+      Three certification levels are defined, named 1, 2, and 3. [TBC: We have a chance to rename, redefine, reorganize, renumber the levels, etc.]
+
+      <section id='certlevels-l1'>
+        <h3>Level 1: Automated testing</h3> 
+
+        Certification at level 1 is intended for verification of a <a>DApp</a> that can be performed by an unguided automated tool. The tool shall be able to determine if the code meets its requirements and is not vulnerable to some vulnerabilities [Discussion: Should we relate to the list of vulnerabilities and define the ones that can/should be dealt at this level, because the tools can do it].
```

Comment:
"unguided" is perhaps misleading in the case of property-based testing, where developers need to define a model to drive the testing, as well as properties over that specific model. I guess there's a general point here: testing can take two forms: entirely generic testing (e.g. double satisfaction) vs DApp-specific testing. In the latter case the developer will need to produce some extra artifacts (tests) which are then subjected to testing, and because they are user-developed there's always that chance that a developer will write them in a way that doesn't completely cover the functionality of the DApp. In short, the quality of the tests has to be assessed by the DApp end user.